### DISCUSSION TITLE
I2C microphone adjustable gain

### DISCUSSION THREAD

**Discussion by honza540** [pschatzmann/arduino-audio-tools#1849] [Q&A] [answered]
Hi, is it possible to adjust the gain of the i2c microphone in aurduino-audio-tool library? The sound level coming out currently is almost not possible to hear (in the web browser player), so the volume needs to be increased on the HW level. It would be great if this library could support it somehow :-)
Thank you and happy New Year :-)

**Comment by pschatzmann** [comment, marked as answer]
There is a wiki for [Volume Control and Panning](https://github.com/pschatzmann/arduino-audio-tools/wiki/Volume-Control)!

If you use a configurable ADC/DAC via the audio-driver library and the ADC supports this, you can use the corresponding setInputVolume() method.
